bin/check-rabbitmq-node-health.rb

```diff
@@ -136,12 +153,26 @@ def run
   end
 
   def node_healthy?
-    host       = config[:host]
-    port       = config[:port]
-    username   = config[:username]
-    password   = config[:password]
-    ssl        = config[:ssl]
-    verify_ssl = config[:verify_ssl_off]
+    host        = config[:host]
+    port        = config[:port]
+    username    = config[:username]
+    password    = config[:password]
+    ssl         = config[:ssl]
+    verify_ssl  = config[:verify_ssl_off]
+    node_prefix = config[:node_prefix]
+
+    # Determine node hostname to query, as this may not be the same as connection hostname
+    node_host = if config[:node_host]
```

Comment:
to address the ABC size I'd move this into its own own function such as `determine_hostname`